django/db/models/functions/math.py

```diff
@@ -158,9 +158,17 @@ def get_group_by_cols(self, alias=None):
         return []
 
 
-class Round(Transform):
+class Round(FixDecimalInputMixin, NumericOutputFieldMixin, Transform):
     function = 'ROUND'
     lookup_name = 'round'
+    arity = None  # Override arity=1 set by Transform to enable passing places.
+
+    def __init__(self, expression, places=None, **extra):
+        # If places is not provided, default to using the single argument form
+        # of the function as supported by all core database backends. All known
+        # backends default to 0 in this case.
+        args = () if places is None else (places,)
+        super().__init__(expression, *args, **extra)
```

Comment:
With `arity = None`, it is possible to ditch this entirely if we default to `places=0`, but I thought it still made sense to use the single argument form of the function as we currently do rather than explicitly passing `0`.